### PR TITLE
INFINITY-3103 Break out template parameters into per-service YAMLs

### PR DESCRIPTION
**Pull request by nickbp**
- Cleans up duplicated template values that were being specified across each service's respective docs sections.
- When docs are being built for docs.mesosphere.com, the respective service's YAML file may be edited in-place before rendering to customize the output. For example, `packageName: beta-kafka` => `packageName: confluent-kafka`.

More info on the data file mechanism: https://jekyllrb.com/docs/datafiles/

### CODE REVIEW

File: frameworks/cassandra/docs/api-reference.md [hunk 1]
```diff
@@ -4,12 +4,7 @@ navigationTitle:
 excerpt:
 title: API Reference
 menuWeight: 70
-
-packageName: beta-cassandra
-serviceName: cassandra
 ---
+{% assign data = site.data.services.cassandra %}
```
Comment (benclarkwood) [inline frameworks/cassandra/docs/api-reference.md]: It isn't possible to centralize this assignment is it?

Comment (nickbp) [reply frameworks/cassandra/docs/api-reference.md]: Not AFAICT, the scopes appear to just be:
- site-wide
- the current page
No separate 'section' scope.

File: docs/pages/_data/services/kafka.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  podType: kafka
+  taskType: broker
+
+supportedVersions:
+  techExampleVersion: 0.11.0.2
+  techLink: "[Apache Kafka](https://kafka.apache.org/downloads)"
+
+# Values specific to Kafka's own docs content:
+
+kafka:
```
Comment (mrbrowning) [inline docs/pages/_data/services/kafka.yml]: What's the value of templating these fields, given that they only apply to this framework?

Comment (nickbp) [reply docs/pages/_data/services/kafka.yml]: Was imagining that they'd need to be changed on-the-fly as well. It is pretty subjective though.

**Pull request merged**